src/likes/likes.service.ts

```diff
@@ -85,7 +91,8 @@ export class LikesService {
                 commentsId
             }
         })
-        return addLike;
+        const action = await this.actionsService.addAction('Create',userId,'CommentLike',addLike.id,addLike)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them